prompt.go

```diff
@@ -126,6 +127,8 @@ func (p *Prompt) feed(b []byte) (shouldExit bool, exec *Exec) {
 		if !completing { // Don't use p.completion.Completing() because it takes double operation when switch to selected=-1.
 			if newBuf, changed := p.history.Newer(p.buf); changed {
 				p.buf = newBuf
+			} else if p.completionOnDown {
+				p.completion.Next()
```

Comment:
Hi @VonC. I'm sorry for my late reply.

It looks this operation should be added to [a `handleCompletionKeyBinding` function](https://github.com/c-bata/go-prompt/blob/3a148d16472911cb8aa6302d7d44c9712a4ab17f/prompt.go#L151-L153) like:


```go
func (p *Prompt) handleCompletionKeyBinding(key Key, completing bool) {
	switch key {
	case Down:
		if completing || p.completionOnDown {
			p.completion.Next()
		}
	case ...
```

Reply:
Good point.
That seems to work too, and is cleaner.

I have updated the PR branch named "arrow". 

I have two other suggestions in my own fork (described in the README), waiting for this one to be completed.